webhook_to_fedora_messaging/migrations/env.py

```diff
@@ -42,7 +44,7 @@ def run_migrations_offline() -> None:
     script output.
 
     """
-    url = str(get_config().database.sqlalchemy.url)
+    url = str(get_url_from_app(create_app))
```

Comment:
There's another line down this file where `get_config()` should be replaced, otherwise things crash.

The docs [recommend](https://sqlalchemy-helpers.readthedocs.io/en/latest/user.html#id1) getting the URL once and setting it in alembic's config object:
```
url = get_url_from_app(create_app)
config.set_main_option("sqlalchemy.url", url)
```

And then you can use the regular alembic template for the `env.py` file downwards. 

Or you could get the URL in a variable in the top-level section, and replace the `engine_from_config` call by a simple `create_engine` call that would use this variable.